app/routes/missions/icecube.mdx

```diff
@@ -5,63 +5,158 @@ handle:
 
 import logo from '~/img/icecube-logo.png'
 
-# IceCube
+# IceCube Neutrino Observatory
 
 <img
   src={logo}
   alt="IceCube logo"
-  width="640"
-  height="181"
+  align="right"
+  width="150"
+  height="135"
   className="height-auto"
 />
 
-**Construction Completion Date:** December 17, 2010
-
-**Planned Lifetime:** IceCube Upgrade planned for Winter 2025
+**Construction Completion Date:** December 2010
 
 **End of Operations:** No specific requirement
 
-**Data Archives:**
-https://icecube.wisc.edu/science/data-releases/
+**Data Archives:**\
+https://dataverse.harvard.edu/dataverse/icecube \
+https://icecube.wisc.edu/science/data-releases/ \
 https://heasarc.gsfc.nasa.gov/W3Browse/icecube/icecubepsc.html
 
-[IceCube](https://icecube.wisc.edu/) is an all-sky detector at the South Pole which detects high energy neutrinos in the range of 10<sup>10</sup> to 10<sup>21</sup> eV. The University of Wisconsin-Madison is the lead management institute for IceCube. The construction of IceCube, maintenance and operations, and the IceCube Upgrade are supported by the National Science Foundation as well as the Department of Energy, the University of Wisconsin Alumni Research Foundation, Michigan State University, and agencies in Belgium, Germany, Sweden, Japan, and Korea.
+The [IceCube Neutrino Observatory](https://icecube.wisc.edu/) is a cubic-kilometer Cherenkov particle detector
+deployed in the Antarctic ice beneath the Amundsen-Scott South Pole Station. It consists of 86 strings of
+photo-detectors, extending to a depth of about 2,500 meters below the glacier's surface and
+instrumenting a cubic-kilometer of ice. The Digital Optical Module photo-detectors detect the light
+produced by relativistic charged particles produced by neutrino interactions in or near the instrumented
+volume of ice.
+
+IceCube is sensitive to neutrinos from all directions (all-sky). Neutrino signatures can be broadly
+classified as track-like, produced by the elongated (~km) emission from secondary muons, or as
+cascade events, compact events (~m) produced electrons or hadronic secondary particles.
+Track events can be reconstructed with an uncertainty of less than 1 degree, while cascade events
+have higher signal purity. IceCube triggers on signals for neutrino energies greater than ~10 GeV
+(10<sup>10</sup> eV) , and can identify likely astrophysical neutrino events for neutrino
+energies greater than ~50 TeV (5 x 10<sup>13</sup> eV) and issue alerts to the community.
```

Comment:
You know what, why don't we just enable math mode for ya?

Reply:
happy to change it if you do, but right now these are the only two exponents or complex math

Reply:
```suggestion
energies greater than ~50 TeV (5×10<sup>13</sup> eV) and issue alerts to the community.
```